pallets/external-validator-slashes/src/mock.rs

```diff
@@ -58,7 +59,7 @@ impl system::Config for Test {
     type BaseCallFilter = frame_support::traits::Everything;
     type BlockWeights = ();
     type BlockLength = ();
-    type DbWeight = ();
+    type DbWeight = RocksDbWeight;
```

Comment:
Without this the cost of db read and writes is 0

Reply:
yep, and this is needed to test, good catch